app/v1/services/DeleteOtherExpensesService.scala

```diff
@@ -40,19 +39,23 @@ class DeleteOtherExpensesService @Inject() (deleteOtherExpensesConnector: Delete
       logContext: EndpointLogContext,
       correlationId: String): Future[DeleteOtherExpensesServiceOutcome] = {
 
-    val result = for {
-      desResponseWrapper <- EitherT(deleteOtherExpensesConnector.deleteOtherExpenses(request)).leftMap(mapDownstreamErrors(desErrorMap))
-    } yield desResponseWrapper
-    result.value
+    deleteOtherExpensesConnector.deleteOtherExpenses(request).map(_.leftMap(mapDownstreamErrors(downstreamErrorMap)))
   }
 
-  private def desErrorMap: Map[String, MtdError] =
-    Map(
+  private val downstreamErrorMap: Map[String, MtdError] = {
+    val errors = Map(
       "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
-      "FORMAT_TAX_YEAR"           -> TaxYearFormatError,
+      "INVALID_TAX_YEAR"          -> TaxYearFormatError,
       "NO_DATA_FOUND"             -> NotFoundError,
+      "INVALID_CORRELATIONID"     -> StandardDownstreamError,
```

Comment:
I think this needs to be added to the Tech Spec too.

Reply:
This is in the old tech spec already but somehow it wasnt added into the code, so I added back in

Reply:
Sorry yeah I was looking at the new one but it shouldn't be in there 🤦